actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -101,17 +104,19 @@ def fresh_when(object = nil, etag: object, last_modified: nil, public: false, te
       head :not_modified if request.fresh?(response)
     end
 
-    # Sets the +etag+ and/or +last_modified+ on the response and checks it against
-    # the client request. If the request doesn't match the options provided, the
-    # request is considered stale and should be generated from scratch. Otherwise,
-    # it's fresh and we don't need to generate anything and a reply of <tt>304 Not Modified</tt> is sent.
+    # Sets the +etag+ and/or +last_modified+ on the response and checks it
+    # against the client request. If the request doesn't match the options
+    # provided, the request is considered stale and should be generated from
+    # scratch. Otherwise, it's fresh and we don't need to generate anything
+    # and a reply of <tt>304 Not Modified</tt> is sent.
```

Comment:
what is the change in this paragraph? Please remove the wrapping if there is no wording change.